### ISSUE TITLE
Radius of Gyration doesn't work with periodic boundaries.

### ISSUE THREAD

**Issue by kain88-de** [MDAnalysis/mdanalysis#1760]
### Expected behaviour

Calculating correct radius of gyration when protein is split on the periodic boundary and we set `pbc=True`

### Actual behaviour

Neither with the pbc option set or without it do I get the correct radius of gyration. This is because to correctly handle the split of a protein on periodic boundaries we would need to recombine proteins that are split (unwrap them)

[Example protein](https://gist.github.com/kain88-de/cca3fe830772931d76ddca24273dfd53) for which the correct value should be 2-4 nm.

### Code to reproduce the behaviour

``` python
import MDAnalysis as mda

u = mda.Universe('protein.gro')
u.atoms.radius_of_gyration(pbc=True)

```

### Currently version of MDAnalysis:
(run `python -c "import MDAnalysis as mda; print(mda.__version__)"`)

**Comment by richardjgowers** [comment]
The `pbc` keyword will apply PBC by forcing everything into the primary unit cell.

I think what you want is `unwrap`, to make the molecule whole and continuous, which is issue #1185 right?

**Comment by kain88-de** [comment]
yeah sorry, unwrap.  Just forcing everything into the primary unit cell is definitely the wrong thing to do for the radius of gyration calculation.

**Comment by orbeckst** [comment]
Should we change what `pbc=True` does for `radius_of_gyration()`? Do we have enough information to always correctly unwrap the selection?

**Comment by orbeckst** [comment]
P.S.: I created the new [label:PBC ](https://github.com/MDAnalysis/mdanalysis/labels/PBC) – we have so many issues relating to periodic boundary conditions – a GSOC student digging into this would be great.

**Comment by richardjgowers** [comment]
I'd rather pbc=True was consistent in all methods (ie packs into unit cell
before calculation) but I agree that for Rgyr you actually want (the as yet
not implemented) unwrap=True kwarg instead.

**Comment by orbeckst** [comment]
Then rather have unwrap=True and ditch useless pbc keyword. Rather be clear than opaque (or somesuch Python Zen).

> On Jan 24, 2018, at 5:28 PM, Richard Gowers <notifications@github.com> wrote:
> 
> I'd rather pbc=True was consistent in all methods (ie packs into unit cell
> before calculation) but I agree that for Rgyr you actually want (the as yet
> not implemented) unwrap=True kwarg instead.

--
Oliver Beckstein
orbeckst@gmail.com * orbeckst@gmx.net

**Comment by richardjgowers** [comment]
I'm going to close this because it's a duplicate of #1185

**Comment by kain88-de** [comment]
I would like to keep this open as the `pbc` keyword doesn't produce a correct result right now as one would expect. The bug should be open until it's removed.
